packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -346,11 +324,9 @@ const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown
                                 )}
                             </Formik>
                         </ThemedScrollbars>
-                        {!is_app_settings && isDesktop() && <ApiTokenArticle />}
+                        {is_desktop && <ApiTokenArticle />}
                     </div>
                 </section>
-                {footer_ref && <ApiTokenFooter />}
-                {overlay_ref && is_overlay_shown && <ApiTokenOverlay />}
```

Comment:
Also not sure why are we removing these and their respective components?

Reply:
@utkarsha-deriv sorry I don't follow. Both of the values are provided from `ApiToken.

Isn't it being rendered here?

<img width="381" alt="Screenshot 2023-08-28 at 12 17 22 PM" src="https://github.com/binary-com/deriv-app/assets/70938039/c9584c50-4ffd-4967-a541-e5c58608dfcf">
Context.Provider` right?